tests/neo4j/sessionrun.py

```diff
@@ -234,6 +234,25 @@ def test_partial_iteration(self):
         self._session1.close()
         self._session1 = None
 
+    def test_simple_query(self):
+        def _test():
+            self._driver.close()
+            self._driver = get_driver(self._backend, userAgent="test")
+            self._session1 = self._driver.session("r", fetchSize=2)
+            result = self._session1.run("UNWIND [1, 2, 3, 4] AS x RETURN x")
+            if consume:
+                result.consume()
```

Comment:
are there assertions we can run on the summary in that case?

Reply:
We could, but there is `tests.stub.summary.test_summary.TestSummary` which is a whole test suite only for making sure that the summary does what it's supposed to.

Reply:
:thinking: but a little assertion that roughly the expected thing happened might be a good idea.